Formula/kubeless.rb

```diff
@@ -2,8 +2,8 @@ class Kubeless < Formula
   desc "Kubernetes Native Serverless Framework"
   homepage "https://github.com/kubeless/kubeless"
   url "https://github.com/kubeless/kubeless.git",
-      :tag => "v0.6.0",
-      :revision => "3bec4b65f29d830bd6709abb07e00f11fe54d646"
+      :tag => "v1.0.0-alpha.1",
```

Comment:
version is parsed as `1.0.0` (our tag-based, as opposed to url-based, version parsing only looks at digits) so would need an explicit 
```
version "1.0.0-alpha.1"
```